resources/mail/layouts/html.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+/* @var $content string Mail contents as view render result */
+?>
+
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
```

Comment:
Perhaps my understanding is a little outdated but ...

- why did you choose this doctype?
- Are spaces allowed before the `XHTML 1.0 strict` doctype?

Reply:
done.